reference/5-api-publisher/Considerations-docker-configuration-and-execution.md

```diff
@@ -0,0 +1,171 @@
+
+![](img/docker/docker-path.png)
+
+Copy env.example file and name it .env. Update the values as desired.
+
+```docker
+# Nuget version
+VERSION=<Package to install from https://dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_artifacts/feed/EdFi/NuGet/EdFi.ApiPublisher/ Eg. 0.0.0-alpha.0.38>
+
+# The section option for apiPublisherSettings file
+BEARER_TOKEN_REFRESH_MINUTES=<Default. 28>
+RETRY_STARTING_DELAY_MILLISECONDS=<Default. 100>
+MAX_RETRY_ATTEMPTS=<Default. 10>
+MAX_DEGREE_OF_PARALLELISM_FOR_RESOURCE_PROCESSING=<Default. 10>
+MAX_DEGREE_OF_PARALLELISM_FOR_POST_RESOURCE_ITEM=<Default. 20>
+MAX_DEGREE_OF_PARALLELISM_FOR_STREAM_RESOURCE_PAGES=<Default. 5>
+STREAMING_PAGES_WAIT_DURATION_SECONDS=<Default. 10>
+STREAMING_PAGE_SIZE=<Default. 75>
+INCLUDE_DESCRIPTORS=<Default. false>
+ERROR_PUBLISHING_BATCH_SIZE=<Default. 25>
+USE_CHANGE_VERSION_PAGING=<Default. false>
+CHANGE_VERSION_PAGING_WINDOW_SIZE=<Default. 25000>
+
+# The file configurationStoreSettings
+PROVIDER=<Could be one of the following values: sqlServer, postgreSql, awsParameterStore, or plainText. Default. plainText>
+SQLSERVER_SERVER=<If PROVIDER is sqlServer this is required Eg. (local)>
+SQLSERVER_DATABASE=<If PROVIDER is sqlServer this is required Eg. EdFi_API_Publisher_Configuration>
+POSTGRESQL_HOST=<If PROVIDER is postgreSql this is required Eg. localhost>
+POSTGRESQL_DATABASE=<If PROVIDER is postgreSql this is required Eg. edfi_api_publisher_configuration>
+AWS_PROFILE=<If PROVIDER is awsParameterStore this is required Eg. default>
+AWS_REGION=<If PROVIDER is awsParameterStore this is required Eg. us-east-1>
+
+# PlainText connections
+SOURCE_NAME=<Name for the source connection Eg. Hosted_Sample_v5.2>
+SOURCE_URL=<Url for the source connection Eg. https://api.ed-fi.org/v5.2/api/>
+SOURCE_KEY=<Key for the source connection Eg. RvcohKz9zHI4>RvcohKz9zHI4
+SOURCE_SECRET=<Secret for the source connection Eg. E1iEFusaNf81xzCxwHfbolkC>
+
+TARGET_NAME=<Name for the target connection Eg. Local_v5.2>
+TARGET_URL=<Url for the target connection Eg. http://localhost:54746/>
+TARGET_KEY=<Key for the target connection Eg. RvcohKz9zHI4>
+TARGET_SECRET=<Secret for the target connection Eg. E1iEFusaNf81xzCxwHfbolkC>
+
+# Logging using Serilog
+WRITE_TO_FILE_PATH=<Path to store the logging file Eg. ../tmp/logs/Ed-Fi-API-PublisherSerilog.log>
+```
+
+Sample .env provide all the different parameters for run ApiPublisher with different configurations. Please provide the information necessary for a specific configuration.
+
+## Change Configuration on .env file
+
+### 2.a. Nuget Version
+
+	Note: If you want to run a different version than the release, you can modify the version in this file using the name of the generated build. (Docker file i.e. ENV VERSION="1.0.1-alpha.0.17")
+![](img/docker/build-version.png)
+
+![](img/docker/build-version-file.png)
+
+### 2.b. Section ApiPublisherSettings file
+
+	If you like, you can change the default parameters or leave them as is.
```

Comment:
Should not have extra space at the beginning of the line.

Reply:
fixed